diff --git a/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_responses.hpp b/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_responses.hpp
index b9be4dea..edcb2f92 100644
--- a/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_responses.hpp
+++ b/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_responses.hpp
@@ -8,6 +8,7 @@
 #include <thread>
 #include <vector>
 
+#include <azure/core/azure_assert.hpp>
 #include <azure/core/operation.hpp>
 #include <azure/core/paged_response.hpp>
 
